src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -308,6 +308,8 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*smoke_TestsROIAlign.*)",
         // Issue: 148527
         R"(.*Snippets.*MatMulTransposeB.*i8.*i8.*)",
+        // Issue: 136881
+        R"(.*smoke_CompareWithRefs_4D_Bitwise.*/EltwiseLayerCPUTest.*_eltwise_op_type=BitwiseLeft.*_model_type=.*(i16|u16|u32).*)",
```

Comment:
So, does the Right shift works for these ETs? It's confusing. The logical right shift should not work. You may need to change the generated data to cover such cases.

Reply:
I've added class to test the Bitwise Shift ops with custom data and more cases.
In general the RightShift doesn't fail for the same edge cases when the sign bit or max value conversion is not involved.